extralit_ocr/extract.py

```diff
@@ -244,15 +243,9 @@ def extract_markdown_with_hierarchy(
 # Convenience / Public API
 # ---------------------------------------------------------------------------
 
```

Comment:
I moved these code from `extralit`'s `extralit-server/src/extralit_server/jobs/pdf_extraction_jobs.py` to here